ui/app/AppLayouts/Onboarding/shared/CreatePasswordModal.qml

```diff
@@ -169,15 +169,15 @@ ModalPopup {
                         passwordValidationError = qsTr("Incorrect password")
                     }
                     else {
-                        Global.applicationWindow.prepareForStoring(repeatPasswordField.text, true)
+                        //Global.applicationWindow.prepareForStoring(repeatPasswordField.text, true)
```

Comment:
We already talked about that but to have it written here.. I think that file can be removed!

Reply:
it seems that this popup is still used in StoreToKeychainSelectionModal.qml with a TODO to replace with the new password view, so I would keep it in for now. I completed the TODO text, mentioning to delete StoreToKeychainSelectionModal.qml as well as CreatePasswordModal.qml if not needed